src/BenchmarkDotNet.Diagnostics.Windows/NativeMemoryDiagnoser.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BenchmarkDotNet.Analysers;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Diagnostics.Windows.Tracing;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Validators;
+using JetBrains.Annotations;
+using Microsoft.Diagnostics.Tracing.Parsers;
+
+namespace BenchmarkDotNet.Diagnostics.Windows
+{
+    public class NativeMemoryDiagnoser : IDiagnoser
+    {
+        private static readonly string LogSeparator = new string('-', 20);
+        internal readonly LogCapture Logger = new LogCapture();
```

Comment:
nit: Logger can be private?

```suggestion
        private readonly LogCapture Logger = new LogCapture();
```